js/ElementRendererProvider.js

```diff
@@ -366,19 +366,21 @@ export default class ElementRendererProvider {
       const defaultPadding = 0;
       const padding = config.padding || defaultPadding;
       let nextLeft = 0;
-      const arrowRight = document.createElement('div');
-      const arrowLeft = document.createElement('div');
+      const arrowRight = document.createElement('button');
+      const arrowLeft = document.createElement('button');
       const divCarouselWrapper = document.createElement('div');
       const carousel = document.createElement('div');
       const carouselOffsetChangedEventName = 'carouselOffsetChange';
       let carouselItemIndex = 0;
       let isRTLDirection = false;
       let cards;
 
-      arrowRight.setAttribute('role', 'button');
+      arrowRight.setAttribute('type', 'button');
```

Comment:
no need for that attribute if the element type is a button

Reply:
@meirotstein If the element is a button, by default it is treated as `type="submit"` so that type is needed

Reply:
gotcha, thanks for clarifying.